app.py

```diff
@@ -230,6 +230,49 @@ def _normalise_post(post):
     post = _embed_post_data(post)
     return post
 
+def _get_team_details(slug):
+    teams = [
```

Comment:
maybe it would here be cleaner to have a external json file `team.json` that you can import directly